extensions/amazon-lambda/maven-archetype/src/main/resources/archetype-resources/src/main/resources/application.properties

```diff
@@ -1 +1,2 @@
-quarkus.lambda.handler=test
\ No newline at end of file
+quarkus.lambda.handler=test
+quarkus.package.type=uber-jar
```

Comment:
Remove uber-jar as default.  Have way to turn off fast-jar

Reply:
I changed it to `legacy` in order to ensure that lambda will continue to work when we switch to `fast-jar`

Reply:
I'll look at the fast-jar format, but, it may be just a matter of building the function.zip differently.  Right now the processor assumes dependencies are in target/lib and creates the function.zip based on that.  I would rather fix this properly than aborting with that exception.  I don't want people upgrading then failing their build with a build exception because they don't have the app.properties set.

Reply:
I'm working on a different PR to upgrade to ApiGatewayHTTPEventv2 for quarkus-amazon-lambda-http.  Should I just do the work in there for this rather than accepting this PR?

Reply:
Sure, that's fine with me